use-cases/using-vpp-and-kata.md

```diff
@@ -0,0 +1,76 @@
+# Setup to run VPP
+
+The Data Plane Development Kit (DPDK) is a set of libraries and drivers for
+fast packet processing. The Vector Packet Processing (VPP) is a platform
+extensible framework which provides out-of-the-box production quality
+switch and router functionality. VPP is a high performance packet-processing
+stack which can run on commodity CPUs. Enabling VPP with DPDK support can
```

Comment:
sub out "which" for "that": stack that can run on commodity CPUs.

Reply:
This as well.